src/plone/restapi/tests/test_documentation.py

```diff
@@ -72,63 +72,61 @@
 UPLOAD_PDF_MIMETYPE = "application/pdf"
 UPLOAD_PDF_FILENAME = "file.pdf"
 
+# How do we open files?
+open_kw = {"newline": "\n"}
 
-def pretty_json(data):
-    return json.dumps(data, sort_keys=True, indent=4, separators=(",", ": "))
 
+def normalize_test_port(value):
+    # When you run these tests in the Plone core development buildout,
+    # the port number is random.  Normalize this to the default port.
+    return re.sub(r"localhost:\d{5}", "localhost:55001", value)
```

Comment:
Now we can remove this line:

https://github.com/plone/plone.restapi/blob/c284b560d1a5cbb75a6bfb780c39fdfd139c4fb9/base.cfg#L74

Reply:
We could remove that yes, if we want to.
@sneridagh You made commit https://github.com/plone/plone.restapi/commit/799e11e9c8d8df1040a7e60fe44ea40af028ce6a two years ago to always use port 55001. Was that to make the http-examples always have the same port? In that case we can now remove this setting, if we want.

Reply:
As long as the documentation `resp` files always get the same value and do not change over builds, it's ok.